genesyscloud/resource_genesyscloud_architect_user_prompt.go

```diff
@@ -1050,6 +1056,20 @@ func flattenPromptResources(promptResources *[]platformclientv2.Promptasset) *sc
 			promptResource["filename"] = t["filename"][0]
 		}
 
+		if schemaResources, ok := d.Get("resources").(*schema.Set); ok {
+			schemaResourcesList := schemaResources.List()
+			for _, r := range schemaResourcesList {
+				if rMap, ok := r.(map[string]interface{}); ok {
```

Comment:
Same again, way to safely cast type. This is similar to the issue Cesar found where an item in a list was nil and not a map because it was defined like this `conditions{}` .